src/fireo/database/database.py

```diff
@@ -51,3 +54,7 @@ def conn(self):
             self._conn = firestore.Client()
         return self._conn
 
+    def list_collections(self):
+        """Returns a list of all collections"""
+        return [ c.id for c in self.conn.collections() ]
```

Comment:
Place the function into `fireo/__init__.py` this **database** class is supposed to be for **Firestore connection** 
You can see other method i.e `transaction`, `batch` etc in `__init__.py` please this method also there.

Reply:
@AxeemHaider that creates a weird relationship since the command does need access to db.   Regardless, is WIP, as I only synced to run travis.